recipe/meta.yaml

```diff
@@ -0,0 +1,78 @@
+{% set name = "intel-extension-for-pytorch" %}
+{% set version = "1.12.1" %}
+
+package:
+  name: {{ name }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/intel/{{ name }}/archive/refs/tags/v{{ version }}00.tar.gz
+  sha256: 6f2b54fc3e0a5e26ccaa0552ba93e6d7ecbdc2902b4920e86fde5efdfe427ebc
+  patches:
+    - patches/0001-add-git-rev.patch
+
+build:
+  number: 0
+  # Anything other than linux-64 is not supported fully by upstream, and 3.11
+  # doesn't have PyTorch yet.
```

Comment:
This link https://intel.github.io/intel-extension-for-pytorch/latest/tutorials/installation.html#install-via-wheel-file will help our future selves

Reply:
Thanks, I added to the comments.